coderd/httpmw/logger_internal_test.go

```diff
@@ -0,0 +1,72 @@
+package httpmw
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/slogtest"
+	"github.com/coder/coder/v2/coderd/tracing"
+)
+
+func TestRequestLoggerContext_WriteLog(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	testLogger := slogtest.Make(t, nil)
```

Comment:
These tests should verify that only one log was written. You could write a fake slog.Sink instead of using slogtest and check if the entry's message is the request method etc.